collect.py

```diff
@@ -115,6 +127,24 @@ def do_something(driver, elem_id=None):
             elem.click()
         elif input_type == 'number':
             elem.send_keys('3')
+        elif input_type == 'search':
+            try:
+                elem.send_keys('marco search')
+                elem.submit()
+            except BaseException:
```

Comment:
We should be more precise with what exception this might throw. What was the exception you've seen?